lib/commonConfig.js

```diff
@@ -34,6 +34,9 @@ var config = {},
     commandRegistry,
```

Comment:
An entry in CHANGES_NEXT_RELEASE about the changes in this PR should be added.

Reply:
Fixed 49512da